src/Components/Interview/Helper/createRecorder.ts

```
@@ -0,0 +1,53 @@
+  recorder.onstop = () => {
+    const blob = new Blob(chunks, { type: mimeType });
+    chunks = [];
+    onStop(blob);
+  };
+
+  recorder.onerror = (event) => {
+    if (onError) onError(new Error(`MediaRecorder error: ${event.error}`));
+  };
+
+  return {
+    start: () => recorder && recorder.start(),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)